cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -791,91 +754,20 @@ func TestValidateCompletedOptions(t *testing.T) {
 				config: secretbootstrap.Config{
 					Secrets: []secretbootstrap.SecretConfig{
 						{
-							From: map[string]secretbootstrap.BitWardenContext{
-								".dockerconfigjson": {
-									DockerConfigJSONData: []secretbootstrap.DockerConfigJSONData{
-										{
-											BWItem:                    "bitwarden-item",
-											RegistryURLBitwardenField: "registryURL",
-											AuthBitwardenAttachment:   "auth",
-											EmailBitwardenField:       "email",
-										},
-										{
-											BWItem:                    "bitwarden-item2",
-											RegistryURLBitwardenField: "registryURL",
-											AuthBitwardenAttachment:   "auth",
-											EmailBitwardenField:       "email",
-										},
-									},
-								},
-							},
-							To: []secretbootstrap.SecretContext{
-								{
-									Cluster:   "default",
-									Name:      "docker-config-json-secret",
-									Namespace: "namespace-1",
-									Type:      "kubernetes.io/dockerconfigjson",
-								},
-							},
-						},
-					},
-				},
-			},
-		},
-		{
-			name: "happy dockerconfigJSON configuration: use RegistryURL",
-			given: options{
-				logLevel: "info",
-				config: secretbootstrap.Config{
-					Secrets: []secretbootstrap.SecretConfig{
-						{
-							From: map[string]secretbootstrap.BitWardenContext{
+							From: map[string]secretbootstrap.ItemContext{
 								".dockerconfigjson": {
 									DockerConfigJSONData: []secretbootstrap.DockerConfigJSONData{
 										{
-											BWItem:                  "bitwarden-item",
-											RegistryURL:             "quay.io",
-											AuthBitwardenAttachment: "auth",
-											EmailBitwardenField:     "email",
-										},
-										{
-											BWItem:                    "bitwarden-item2",
-											RegistryURLBitwardenField: "registryURL",
-											AuthBitwardenAttachment:   "auth",
-											EmailBitwardenField:       "email",
+											Item:        "item-1",
+											RegistryURL: "test.com",
+											AuthField:   "auth",
+											EmailField:  "email",
 										},
-									},
-								},
-							},
-							To: []secretbootstrap.SecretContext{
-								{
-									Cluster:   "default",
-									Name:      "docker-config-json-secret",
-									Namespace: "namespace-1",
-									Type:      "kubernetes.io/dockerconfigjson",
-								},
-							},
-						},
-					},
-				},
-			},
-		},
-		{
-			name: "sad dockerconfigJSON configuration: cannot set both RegistryURL and RegistryURLBitwardenField",
```

Comment:
Why was this test removed?

Reply:
`RegistryURLBitwardenField` is removed, so there is no need for that validation at all.